src/components/users/UserProfile.js

```diff
@@ -73,10 +73,16 @@ const UserProfile = ({
         {loggingOut && <CustomLoader />}
       </div>
       <hr className="user-profile__hr" />
-      {seeMatches ? (
-        <UserMatches userMatches={userMatches} />
-      ) : (
-        <TargetsList userTargets={userTargets} setMapCenter={setMapCenter} />
+      {seeMatches
```

Comment:
From my perspective ask first for requestError and then for seeMatches will improve code readability and also decrease the number of time you need to evaluate props